src/storages/Storage.js

```diff
@@ -0,0 +1,16 @@
+function Storage(key) {
```

Comment:
Great, you can also use [ES6 classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes).